package.json

```diff
@@ -1,7 +1,7 @@
 {
-  "name": "kinvey-backend-sdk",
+  "name": "kinvey-flex-sdk",
   "version": "0.5.1",
-  "description": "Backend SDK",
+  "description": "SDK for creating Kinvey Flex Services",
   "engines": {
     "node": "= 6.2.x"
```

Comment:
We should kill two birds with one stone and set/test this against node 6.9.1 since there's currently a PR against the phusion_v6 base image to use node 6.9.1